draft-deepspace-ip-architecture.xml

```diff
@@ -216,6 +239,23 @@
+          <li><t>Congestion Control - Internet congestion control algorithms are based on timely receipt of feedback in order to detect losses, delays, or other signals, and typically attempt to react rapidly.  Two challenges in the interplanetary case include: (1) The relative delays on interplanetary paths are much longer (preventing "rapid" response) and may vary in orders of magnitude between segments, e.g. across a terrestrial segment, an Earth-Mars segment, and a Mars orbit/surface segment.  Congestion may occur in low-latency portions of the network, but fail to be detected quickly because loss/acknowledgement information propagates and feeds back too slowly, leading to sustained loss due to unresponsiveness in a low-latency portion of the network.  (2) By the time feedback is received, it is badly out of date, if not completely irrelevant to the future.  The advent of IP-based in-network storage for scheduled interplanetary links also changes the way that congestion can be measured (e.g. in delay-based protocols).  In the near-term, and in present space mission operations, it can be assumed that data link capacity is explicitly planned and scheduled end-to-end in order to accomodate mission needs.  This makes congestion control algorithms largely replaceable with simple nominal rate and flow control.  However, for larger and more dynamic future interplanetary networks, and shared trunk links, etc., it will be beneficial to develop more appropriate congestion control methods, including possibly more effective network-based signaling/feedback, rather than simply end-to-end feedback-based mechanisms.</t></li>
+          <li><t>Path MTU Discovery - Internet transports typically include probing algorithms and rely on some combination of ICMP messages  from the network and/or end-to-end acknowledgements in order to infer the maximum packet sizes that can be used at any time without introducing unnecessary IP fragmentation.  Because for interplanetary cases, feedback may be slow or unrelated to the current or future paths by the time it arrives, it may be more beneficial to simply rely on known / pre-arranged path MTU limitations that can be communicated between interplanetary providers and other connected providers and users.  Especially for higher bandwidth links, it may be beneficial for this to be significantly larger than the minimum Internet MTU values that are normally assumed.  For instance, terrestrially many systems use 9000+ byte MTUs.  MTUs in interplanetary networks may vary due to the range of link bandwidths, and simultaneous desires to be efficient on fast links while also wanting to avoid head-of-line blocking for large packets on low-bandwidth links.</t></li>
```

Comment:
Actually, ICMP-based  path mtu discovery is not reliable because of being filtered. I think we should instead talk essentially about transport-based path MTU discovery, such as the one in QUIC.